### ISSUE TITLE
feat(remote): configure game indexing scope

### ISSUE THREAD

**Issue by RGarciaLago** [wizzomafizzo/mrext#11]
Allow Remote users to choose which systems are included when generating `search.db`.

The indexing scope should also support excluded paths, including `_Arcade/_alternatives`, so duplicate or unwanted entries can be omitted.

**Comment by ransom1122** [comment]
I agree, when experimenting with one platform, it'll help a lot if you could only scan one platform, instead of "scan all" all the time.

**Comment by wizzomafizzo** [comment]
This remains a valid feature request, but it would expand Remote’s local indexing UI, configuration, and persistence requirements. Current mrext maintenance is focused on keeping existing behavior stable rather than adding new Remote features. Closing as not planned.
